### PR TITLE
Command-line interface for compressing and decompressing resource timing data

### PR DESCRIPTION
**Pull request by msolnit**
…ing data.

### CODE REVIEW

**Comment by nicjansma** [comment]
Can you make sure to add `commander` to `package.json`?

**Comment by msolnit** [comment]
Sure.  The version I have on my machine is 2.8.1.  Does that work for you?

**Comment by nicjansma** [comment]
Yeah, whatever's latest -- you can just do `npm install commander --save-dev` and it'll add it automagically.

File: cli/decompress.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+module.exports = function(inputFile, options) {
+    var rt = JSON.parse(fs.readFileSync(inputFile));
+    var decompressed = ResourceTimingDecompression.decompressResources(rt);
+
+    var space;
+    var outputFile;
+
+    if (options && options.parent) {
+        if (options.parent.pretty)
+            space = 2;
```
Comment (nicjansma) [inline cli/decompress.js]: Always use braces please (I think `grunt lint` should catch this?).

File: cli/compress.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+//
+module.exports = function(inputFile, options) {
+    var entries = JSON.parse(fs.readFileSync(inputFile));
+    var compressed = ResourceTimingCompression.compressResourceTiming(entries);
+
+    var space;
+    var outputFile;
+
+    if (options && options.parent) {
+        if (options.parent.pretty)
```
Comment (nicjansma) [inline cli/compress.js]: Always use braces please (I think `grunt lint` should catch this?).

File: cli/decompress.js (hunk 1 above)
Comment (msolnit) [inline cli/decompress.js]: There's no `Gruntfile.js` ;-).

**Comment by nicjansma** [comment]
My bad, I forgot this is using gulp.

**Comment by nicjansma** [comment]
Thanks!

**Pull request merged**